scripts/src/errors.ts

```diff
@@ -46,7 +46,8 @@ const CODES = {
 		MaxWalletsExceeded: 7,
 		InvalidWalletAddress: 8,
 		ExpiredJwt: 9,
-		InvalidJwtIssuedTime: 10
+		InvalidJwtIssuedTime: 10,
+		MissingFieldJWT: 11
```

Comment:
Add to error-cods wiki

Reply:
here: https://github.com/kinecosystem/marketplace-server/wiki/Error-Codes

Reply:
Added